administrator/components/com_finder/src/Model/IndexerModel.php

```diff
@@ -11,13 +11,38 @@
 
 \defined('_JEXEC') or die;
 
-use Joomla\CMS\MVC\Model\BaseDatabaseModel;
+use Joomla\CMS\Form\Form;
+use Joomla\CMS\MVC\Model\FormModel;
 
 /**
  * Indexer model class for Finder.
  *
  * @since  2.5
  */
-class IndexerModel extends BaseDatabaseModel
+class IndexerModel extends FormModel
 {
+	/**
+	 * Method for getting a form.
+	 *
+	 * @param   array    $data      Data for the form.
+	 * @param   boolean  $loadData  True if the form is to load its own data (default case), false if not.
+	 *
+	 * @return  Form
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 *
+	 * @throws \Exception
+	 */
+	public function getForm($data = array(), $loadData = true)
```

Comment:
Use short array syntax throughout

```suggestion
	public function getForm($data = [], $loadData = true)
```